### PR TITLE
feat: [ASL-4532] Add redux dev tools

### PR DESCRIPTION
**Pull request by kiran-varma-home-office**

### CODE REVIEW

**Review by jeff-horton-ho-sas — APPROVED** [review]

File: client/store.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 import thunk from 'redux-thunk';
+import { composeWithDevTools } from '@redux-devtools/extension';
 
 const middlewares = [thunk];
 
 export default function configureStore(initialState = {}) {
+
+  let middleware;
+
+  if (process.env.NODE_ENV === 'development') {
```
Comment (farrukhishaq) [inline client/store.js:12]: new entry to `.env.example`
NODE_ENV === 'development'

**Review by farrukhishaq — COMMENTED** [review]
Looks good.

File: client/store.js (hunk 1 above)
Comment (kiran-varma-home-office) [reply client/store.js:12]: that's in different repo right? - asl-internal-ui

I guess I raise another PR for it?

Comment (jeff-horton-ho-sas) [reply client/store.js:12]: There will need to be a cascade of PRs - so after this there will need to be a commit to asl-pages bumping the asl-projects version, then to asl and asl-internal-ui bumping the pages and projects versions. You can update the .example.env at the same time.

**Pull request merged**